src/sympc/tensor/mpc_tensor.py

```diff
@@ -426,7 +426,7 @@ def conv2d(
         convolution = self.__apply_op(weight, "conv2d", kwargs_=kwargs)
 
         if bias:
-            return convolution + bias
+            return convolution + bias.unsqueeze(1).unsqueeze(1)
```

Comment:
Are you sure the double unsqueeze is needed?

Reply:
There was a problem here because the bias was not broadcasted accordingly :(
Bias was having the shape [10] and I think the convolution was something like [10, X, X] where X was a number.

Reply:
Hmmm that's strange! But ok let's keep it as such for now